Source/ActorPool/Private/ActorPoolActor.cpp

```diff
@@ -192,7 +201,10 @@ void AActorPoolActor::BeginPlay()
     {
         for ( const auto & pool_infos : settings->PoolInfos )
         {
-            ActorPools.Emplace( pool_infos.ActorClass.LoadSynchronous(), CreateActorPoolInstance( pool_infos ) );
+            if ( HasAuthority() && pool_infos.bSpawnOnServer || !HasAuthority() && pool_infos.bSpawnOnClients )

```

Comment:
Proper check would be something like:

```
const auto is_standalone = UKismetSystemLibrary::IsStandalone( world );
    auto is_server = IsRunningDedicatedServer();

#if WITH_EDITOR
    checkSlow( game_instance->GetWorldContext() );
    is_server |= world->GetGameInstance()->GetWorldContext()->RunAsDedicated;
#endif

    const auto is_client = !is_server;

    if ( is_server && pool_infos.bSpawnOnServer || is_client && pool_infos.bSpawnOnClients )
```

Check `UGFEGameFeatureAction_SpawnActors`

Reply:
Done.